src/easier68k/core/util/conversions.py

```diff
@@ -1,39 +1,38 @@
 # Conversion utils
 
 
-def word_to_long(word: int) -> int:
+def to_word(word: int) -> int:
     """
-    >>> hex(word_to_long(0x12341234))
+    >>> hex(to_word(0x12341234))
     '0x1234'
 
-    >>> hex(word_to_long(0))
+    >>> hex(to_word(0))
     '0x0'
 
     This size should never happen!
-    >>> hex(word_to_long(0x123412341))
+    >>> hex(to_word(0x123412341))
     '0x2341'
 
-    Masks the given word value to a long
-    Also will work with bytes and words
+    Masks the given value to the size of a word (16 bits)
```

Comment:
Do we want to have a different version that instead of masking does bit extension instead? I know that's relevant in some places.

Reply:
I'm not certain what is meant by this, do you mean something like this: `0xAB` -> `0x000000AB` ? Because if so, that's already being done, `hex(` doesn't show the leading zeros.